app/src/main/res/values/colors.xml

```diff
@@ -3,5 +3,9 @@
   <color name="colorPrimary">#008577</color>
   <color name="colorPrimaryDark">#00574B</color>
   <color name="colorAccent">#D81B60</color>
-  <color name="ic_launcher_background">#26A69A</color>
+  <color name="lightGreen">#26A69A</color>
+  <color name="red">#C55F45</color>
```

Comment:
This isn't true red. For this & all other colors being introduce here, please provide context for where they're being used (e.g. 'highlighted_nav_menu_item' or something similar).

Reply:
resolved